File: common/src/main/java/org/valkyrienskies/clockwork/ClockWorkTab.java
package org.valkyrienskies.clockwork;

import com.simibubi.create.infrastructure.item.CreateCreativeModeTab;
import net.minecraft.world.item.ItemStack;

public class ClockWorkTab extends CreateCreativeModeTab {
    public ClockWorkTab() {
        super("clockwork");
    }

    @Override
    public ItemStack makeIcon() {
        return ClockWorkBlocks.PHYSICS_INFUSER.asStack();
    }
}

File: common/src/main/kotlin/org/valkyrienskies/clockwork/ClockworkTab.kt
package org.valkyrienskies.clockwork

import com.simibubi.create.infrastructure.item.CreateCreativeModeTab
import net.minecraft.world.item.ItemStack

class ClockworkTab : CreateCreativeModeTab("clockwork") {
    override fun makeIcon(): ItemStack {
        return ClockworkBlocks.PHYSICS_INFUSER.asStack()
    }
}
